ballerina/file_string_io.bal

```diff
@@ -131,8 +131,8 @@ public isolated function fileWriteJson(string path, json content) returns Error?
 # ```
 # + path - The path of the XML file
 # + content - XML content to write
-# + xmlOptions - XML writing options(XML entity type and DOCTYPE)
-# + fileWriteOption - file write option(`OVERWRITE` and `APPEND` are the possible values, and the default value is `OVERWRITE`)
+# + xmlOptions - XML writing options (XML entity type and DOCTYPE)
+# + fileWriteOption - File write option (`OVERWRITE` and `APPEND` are the possible values, and the default value is `OVERWRITE`)
```

Comment:
```suggestion
# + fileWriteOption - File write option (`OVERWRITE` and `APPEND` are the possible values and the default value is `OVERWRITE`)
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/477/commits/b940cf5754398095dd38803252c05359a666001c